cmd/tendermint/commands/testnet.go

```diff
@@ -73,6 +77,23 @@ Example:
 
 func testnetFiles(cmd *cobra.Command, args []string) error {
 	config := cfg.DefaultConfig()
+
+	// overwrite default config if set and valid
+	if configFile != "" {
+		base := &cfg.Config{}
+		viper.SetConfigFile(configFile)
+		if err := viper.ReadInConfig(); err != nil {
+			return err
+		}
+		if err := viper.Unmarshal(base); err != nil {
```

Comment:
maybe use `config` here? that's what we do in cmd/tendermint/root.go#ParseConfig

Reply:
Done :)